src/include/com/amazonaws/kinesis/video/webrtcclient/Include.h

```diff
@@ -318,6 +320,12 @@ extern "C" {
  */
 #define MAX_MESSAGE_DESCRIPTION_LEN                                                 1024
 
+
+/**
+ * Max certificates an RtcConfiguration can accept
+ */
+#define MAX_RTCCONFIGURATION_CERTIFICATES                                           3
```

Comment:
* How long these certs are valid?
* How this is handled for concurrent sessions?

Reply:
These are passed by the customer. If they are invalid then the DTLS will fail to handshake. A certificate can be used to establish multiple sessions - a symmetric ephemeral session key is derived

Reply:
- How is this exposed to the customer?
- Do we have a go ahead from security?

Reply:
The customer can specify set of cert/keys in the rtc config when creating peer connection. If the cert fails the validity an error will be returned in peer create API. The srtp which establishes DTLS sitting on top of open ssl which, in turn, uses lib crypto will do the cipher suite negotiation, etc. For now, we are not pushing this to master, though, I don't see any issues.